app/views/shut_mucr_confirmation.scala.html

```diff
@@ -22,17 +22,22 @@
 
 @()(implicit request: Request[_], flash: Flash, messages: Messages)
 
+@shutMucrLink = {<a href="@routes.ChoiceController.startSpecificJourney("shutMUCR")">@messages("shutMucr.confirmation.nextSteps.shutMucr")</a>}
```

Comment:
Can we use variable as parametr to reverse routing?

Reply:
Yes. I'll do it in the next PR to merge those changes.